libraries/rush-lib/src/cli/actions/test/removeRepo/.gitignore

```diff
@@ -0,0 +1 @@
+common/temp
```

Comment:
This should already be ignored. All folders in the repo called `temp` are ignored.

Reply:
remove this file